### ISSUE TITLE
Local private bot

### ISSUE THREAD

**Issue by smooreace** [matnad/pollmaster#24]
Is it possible to run those localized instead of utilizing a public bot? Not that there is any problem. But I like to run bots that I have control of rather than depend on someone else.

**Comment by matnad** [comment]
Sorry for the late response. Yes it is absolutely possible. I will write a short doc on how to do it, but its not very hard to figure out (just clone/fork the repo and add the secrets file locally).
If you are having troubles with it feel free to join the support discord and tag me.

**Comment by smooreace** [comment]
Great,,, will do Thanks

**Comment by matnad** [comment]
The secrets file should look like this: 

```
class Secrets:
    def __init__(self):
        self.dbl_token = ''  # DBL token (only needed for public bot)
        self.mongo_db = 'mongodb://localhost:27017/pollmaster'
        self.bot_token = '' # official discord bot token
        self.mode = 'development' # or production

SECRETS = Secrets()
```

And you need to set up a MongoDB.

**Comment by matnad** [comment]
fixed by #29
